ndc-reference/tests/query/nested_array_select/request.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "../../../../ndc-client/tests/json_schema/query_request.jsonschema",
+    "collection": "universities",
+    "arguments": {},
+    "query": {
+        "fields": {
+            "id": {
+                "type": "column",
+                "column": "id"
+            },
+            "professors": {
+                "type": "column",
+                "column": "professors",
+                "nested_field": {
```

Comment:
Can we call this `fields` (and the same for `array`) to be consistent everywhere?

Reply:
Done